src/pages/VoterDetail/History/VoterProposal/index.tsx

```diff
@@ -0,0 +1,215 @@
+              components={{
+                Span: <Typography color="textPrimary" />,
+              }}
+              values={{
+                date: endDate,
+              }}
+            />
+          ),
+        ];
+      case 'Queued':
+        return [
+          <InactiveChip text={t('voteProposalUi.proposalState.queued')} />,
+          queuedDate && (
+            <Trans
+              i18nKey="voteProposalUi.proposalState.queuedTimestamp"
+              components={{
+                Span: <Typography color="textPrimary" />,
+              }}
+              values={{
+                date: queuedDate,
+              }}
+            />
+          ),
+        ];
+      case 'Defeated':
+        return [
+          <ErrorChip text={t('voteProposalUi.proposalState.defeated')} />,
+          endDate && (
+            <Trans
+              i18nKey="voteProposalUi.proposalState.defeatedTimestamp"
+              components={{
+                Span: <Typography color="textPrimary" />,
+              }}
+              values={{
+                date: endDate,
+              }}
+            />
+          ),
+        ];
+      case 'Executed':
+        return [
+          <BlueChip text={t('voteProposalUi.proposalState.executed')} />,
+          executedDate && (
+            <Trans
+              i18nKey="voteProposalUi.proposalState.executedTimestamp"
+              components={{
+                Span: <Typography variant="small2" color="textPrimary" />,
+              }}
+              values={{
+                date: executedDate,
+              }}
+            />
+          ),
+        ];
+      default:
+        return [];
+    }
+  }, [proposalState]);
+
+  return (
+    <Paper
+      className={className}
+      css={styles.root}
+      component={({ children, ...props }) => (
+        <div {...props}>
+          <Link to={Path.VOTE_PROPOSAL_DETAILS.replace(':id', proposalNumber.toString())}>
+            {children}
+          </Link>
+        </div>
+      )}
+    >
+      <Grid container>
+        <Grid css={[styles.gridItem, styles.gridItemLeft]} item xs={12} sm={8}>
+          <div css={styles.cardHeader}>
+            <div>
+              <Chip text={`#${proposalNumber}`} />
+              {stateChip}
+            </div>
+
+            {voteChip}
+          </div>
+
+          <Typography variant="h4" css={styles.cardTitle} color="textPrimary">
+            {proposalTitle}
+          </Typography>
+
+          <Typography variant="small2" component="span">
+            {stateTimestamp}
+          </Typography>
+        </Grid>
+        <Grid css={[styles.gridItem, styles.gridItemRight]} item xs={12} sm={4}>
+          <ActiveVotingProgress
+            votedForWei={forVotesWei}
+            votedAgainstWei={againstVotesWei}
+            abstainedWei={abstainedVotesWei}
+            votedTotalWei={votedTotalWei}
+          />
+        </Grid>
+      </Grid>
+    </Paper>
+  );
+};
+
+export default GovernanceProposal;
```

Comment:
This component is a duplicate of `GovernanceProposal`. Since the UI of these two components is similar with a few modifications we should create a reusable component to share between the Vote page and the Voter Details page.